test/sales_engine_test.rb

```diff
@@ -17,10 +18,14 @@ def test_creates_instance_of_merchant_repository
     assert_instance_of MerchantRepository, @se.merchants
   end
 
+  def test_creates_instance_of_invoice_repository
```

Comment:
maybe rename to test_it_creates or test_it_has; probably doesn't matter much though - it's still clear

Reply:
Will do. Good to be consistent!